### DISCUSSION TITLE
Cut function missing?

### DISCUSSION THREAD

**Discussion by Augster1** [OrcaSlicer/OrcaSlicer#5863] [Q&A] [answered]
I'm on the latest version (2.1.0) and noticed the Cut function isn't showing at the to of the prep screen. Anyone else missing it?

![Orca screenshot](https://github.com/SoftFever/OrcaSlicer/assets/173857431/c9d47ad4-3b18-4a1c-a6d3-d8987e08f886)

**Comment by Noisyfox** [comment, marked as answer]
Have you enabled advanced mode?

**Reply by Augster1** [reply]
That was it! I just installed Orca on a new machine and hadn't enabled Adv. setting yet. Thanks for the quick response!

**Comment by Avpman2** [comment]
O.T. but - I have a 3d model of a square frame to raise the cover on an ADM5Pro. The model is too big to print as a single piece on the 5M. I have tried many methods to cut the frame into four pieces I could print and later glue together with ZERO luck. Would you, or someone, be willing to cut up the model for me? I'll post it here if there are any bites. I would sincerely appreciate any help.
